Mathlib/Data/Real/Sqrt.lean

```diff
@@ -103,7 +103,7 @@ namespace Real
 /-- The square root of a real number. This returns 0 for negative inputs.
 
 This has notation `√x`. Note that `√x⁻¹` is parsed as `√(x⁻¹)`. -/
-noncomputable def sqrt (x : ℝ) : ℝ :=
+@[irreducible] noncomputable def sqrt (x : ℝ) : ℝ :=
```

Comment:
Does it make sense to make `NNReal.sqrt` irreducible instead? I think having the two sqrts defeq is handy.

Reply:
I'm trying that now.

Reply:
But note:
```lean
import Mathlib.Data.Complex.Norm

lemma test {α : Type*} [Norm α] (a b : α) : ‖a‖ = ‖b‖ := sorry

variable (x : ℝ) (z : ℂ)

-- #count_heartbeats in -- Used 217827 heartbeats
-- #check (test (↑x) z : ‖Complex.ofReal x‖ = ‖z‖)

seal Real.sqrt in
#count_heartbeats in -- Used 114 heartbeats
#check (test (↑x) z : ‖Complex.ofReal x‖ = ‖z‖)

seal NNReal.sqrt in
#count_heartbeats in -- Used 3353 heartbeats
#check (test (↑x) z : ‖Complex.ofReal x‖ = ‖z‖)
```